commands/station.js

```diff
@@ -27,13 +28,26 @@ export const station = async () => {
 
   startPingLoop().unref()
 
-  await saturnNode.start({
-    FIL_WALLET_ADDRESS,
-    MAX_DISK_SPACE,
-    storagePath: join(paths.moduleCache, 'saturn-L2-node'),
-    binariesPath: paths.moduleBinaries,
-    metricsStream: createMetricsStream(paths.metrics),
-    activityStream: createActivityStream('Saturn'),
-    logStream: createLogStream(join(paths.moduleLogs, 'saturn-L2-node.log'))
-  })
+  await Promise.all([
+    saturnNode.start({
+      FIL_WALLET_ADDRESS,
+      MAX_DISK_SPACE,
+      storagePath: join(paths.moduleCache, 'saturn-L2-node'),
+      binariesPath: paths.moduleBinaries,
+      metricsStream: createMetricsStream(paths.metrics),
+      activityStream: createActivityStream('Saturn'),
+      logStream: createLogStream(join(paths.moduleLogs, 'saturn-L2-node.log'))
+    }),
+
+    zinniaRuntime.start({
+      FIL_WALLET_ADDRESS,
+      STATE_ROOT: join(paths.moduleState, 'zinnia'),
+      CACHE_ROOT: join(paths.moduleCache, 'zinnia'),
+      moduleBinaries: paths.moduleBinaries,
+      // FIXME(bajtos) We need to merge metrics reported by different modules
+      metricsStream: createMetricsStream(paths.metrics.replace(/\.log$/, '-zinnia.log')),
```

Comment:
This needs some refactoring from me: There should only be one metrics file, so the stats of all modules writing to it need to merged. Will work on it

Reply:
Or rather: each module writes its own metrics file, and the `station metrics` command merges them

Reply:
https://github.com/filecoin-station/core/pull/94

Reply:
With that merged in, the current line can be changed to this:

```suggestion
      metricsStream: createMetricsStream('zinnia'),
```

Reply:
Once `main` has been merged in, the snippet from the last comment should now work

Reply:
Awesome, thank you!